content/en/blog/2022/announcing-community-manager.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Announcing the OpenTelemetry Community Manager
+linkTitle: Announcing OpenTelemetry Community Manager
+date: 2022-09-24
+author: Austin Parker
+---
+
+OpenTelemetry has demonstrated massive growth since its inception in 2019. What started as a handful of OpenTracing and OpenCensus maintainers and collaborators meeting at the Google campus and over Zoom has now grown into the second-most popular project in the CNCF behind Kubernetes itself. Over 5400 contributors and 700 companies have contributed code, issues, documentation, and invaluable feedback. 
+
+Our community isn’t just composed of these contributors, however - our end-users, partners, integrators, and a whole host of other people make up the OpenTelemetry community. In order to serve the needs of the wider end-user community, the governance committee has formed a community manager role who can focus on the needs of this large, and growing, group of humans.
+
+Community managers are appointed by the Governance Committee and act as public stewards of the contributor and end-user community. They’re responsible for organizing events, coordinating cross-SIG and WG efforts to improve contributor experience, managing the OpenTelemetry presence on social media, and working to nurture and grow the OpenTelemetry community overall. Effectively, this is a new project maintainer whose project is the OpenTelemetry community itself.
+
+With that said, I’m happy to inform you that I have been designated the first Community Manager for the OpenTelemetry project! As a former OpenTracing maintainer, I’ve been a part of OpenTelemetry since its inception, mostly working on projects such as the website, hosting OpenTelemetry Tuesdays, and helping organize events like OpenTelemetry Community Day. With this new role, I plan to more formally establish OpenTelemetry Community Days around the world as part of KubeCon/CloudNativeCon -- and additionally, help promote new end-user and contributor events like OTel Unplugged (more on that in a week or so.)
```

Comment:
Might be cool to have a picture of your face or something here to add a 'lil more human element to it

Reply:
eh it'd be weird if it was just a big picture of me in there.